### DISCUSSION TITLE
Authenticate with Auth0 does not work if Redirect HTTP requests to HTTPS is enabled

### DISCUSSION THREAD

**Discussion by pcolmer** [requarks/wiki#3614] [Error / Bug Report]
**Describe the bug**
Attempting to log in with Auth0 results in a page saying

Oops, something went wrong...
Invalid email / username or password.

**To Reproduce**
1. Configure Wiki.js to support Auth0 authentication.
2. Enable "Redirect HTTP requests to HTTPS".
3. Try to authenticate with Auth0.

**Expected behavior**
Auth0 authentication should work, creating a user if required.

**Screenshots**
![image](https://user-images.githubusercontent.com/1979931/94405165-124a0700-0168-11eb-89cc-4d8e0843f797.png)

**Host Info (please complete the following information):**
 - OS: AWS AMI
 - Wiki.js version: 2.5.144
 - Database engine: postgres:11

**Additional context**
Even when "Redirect HTTP requests to HTTPS" is enabled, the Auth0 configuration says that the callback starts with "http://", not "https://". Auth0 says:

"Make sure to specify the protocol (https://) otherwise the callback may fail in some cases. With the exception of custom URI schemes for native clients, all callbacks should use protocol https://."

The "Oops, something went wrong" message is generated from the callback URL - it is generating a 500 Internal Server Error.

Turning off "Redirect HTTP requests to HTTPS" allows authentication to work but then browsers report the wiki pages as insecure.

**Comment by sickmeyer** [comment]
Hello. Any updates on this?
